Extension/YetAnotherPartyOrganiser.csproj

```diff
@@ -62,14 +62,16 @@
         <WarningLevel>4</WarningLevel>
     </PropertyGroup>
     <ItemGroup>
-        <Reference Include="0Harmony, Version=2.0.0.9, Culture=neutral, PublicKeyToken=null">
-          <HintPath>packages\Lib.Harmony.2.0.0.9\lib\net472\0Harmony.dll</HintPath>
-          <Private>True</Private>
+        <Reference Include="0Harmony, Version=2.0.0.10, Culture=neutral, PublicKeyToken=null">
+            <HintPath>packages\Lib.Harmony.2.0.0.10\lib\net472\0Harmony.dll</HintPath>
+            <Private>True</Private>
+        </Reference>
+        <Reference Include="0UIExtenderLibModule, Version=1.0.5.0, Culture=neutral, PublicKeyToken=null">
```

Comment:
This is definitely wrong since it uses a relative path.

Reply:
Will be resolved when harmony update is on master and merged into this branch